tpl/bootstrap.php

```diff
@@ -578,7 +578,7 @@
 					<h4 class="col-md-5 col-xs-8"><?php echo I18n::_($NAME); ?> <small>- <?php echo I18n::_('Because ignorance is bliss'); ?></small></h4>
 					<p class="col-md-1 col-xs-4 text-center"><?php echo $VERSION; ?></p>
 					<p id="aboutbox" class="col-md-6 col-xs-12">
-						<?php echo I18n::_('%s is a minimalist, open source online pastebin where the server has zero knowledge of pasted data. Data is encrypted/decrypted <i>in the browser</i> using 256 bits AES. More information on the <a href="https://privatebin.info/">project page</a>.', I18n::_($NAME)), PHP_EOL; ?>
+						<?php echo I18n::_('%s is a minimalist, open source online pastebin where the server has zero knowledge of pasted data. Data is encrypted/decrypted <i>in the browser</i> using 256 bits AES. %s', I18n::_($NAME)), I18n::_($INFO), PHP_EOL; ?>
```

Comment:
Either you need to remove the suffix " %s" from the translation strings (my preference) or you keep it, but then need to remove the ending bracket after `I18n::_($NAME)`, so that three strings are passed as arguments to the first `I18n::_()` call.

Reply:
Fixed. Thanks